<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family:century gothic ;
font-size:24px;


}
QMainWindow{
background:white;
border-radius:15px ;
}
QFrame{
background-color:#a9a8ff;
border-radius:15px ;
}

#loginWidgets{
background:white;
border-radius:15px ;
box-shadow: 10px 10px;
}

QToolButton{
background:white;
border-radius:60px;
}
QLabel{
color:black;
}
QPushButton{
color:#918FFF;
background:white;
border-radius:15px ;
}
QPushButton:hover
{
color:#6563FF;
background:#EBEBEB;
border-radius:15px ;
}
QLineEdit{
background:transparent;
border-radius:14px;
border:none;
color:white;
border-bottom:1px solid white;
}
QLineEdit:hover{
color:black;
border-bottom:1px solid black;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>601</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/login res/loggooo.png);
border-radius:15px;
  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QStackedWidget" name="loginWidgets">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>591</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>531</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>400</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Sign up</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>400</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="username_line">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>230</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>User name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="password_line">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>310</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>130</y>
         <width>171</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size:40px;
font-weight: 700;
color:white;</string>
       </property>
       <property name="text">
        <string>Log in</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>130</y>
         <width>171</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size:40px;
font-weight: 700;
color:white;</string>
       </property>
       <property name="text">
        <string>Sign up</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="username_signup_line">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>230</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>User name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="password_signup_line">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>310</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>400</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Sign up</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>400</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/feather (3)/arrow-left.svg</normaloff>:/feather (3)/arrow-left.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1210</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
